services/QuillLMS/engines/evidence/app/models/evidence/labeled_entry.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+# == Schema Information
+#
+# Table name: evidence_labeled_entries
+#
+#  id         :bigint           not null, primary key
+#  embedding  :vector(1536)     not null
+#  entry      :text             not null
+#  label      :text             not null
+#  metadata   :jsonb
```

Comment:
So, I think we'll also want another nullable boolean field in here called `approved` or something like that. 

Basically, it would be nice to have a place to store the entry/embedding from the incoming student responses (so we only need to fetch from the API once), but only use the embeddings that have been flagged as `approved` (as whatever name) for the nearest neighbor analysis. Thinking a nullable boolean since 'null' would be "unlooked at", 'false' would "this is incorrect, ignore it for now". Happy to discuss, especially if you have different ideas of what to do with incoming data (I don' t have strong opinions here).